pkg/libratopublisher/publisher.go

```diff
@@ -0,0 +1,37 @@
+package libratopublisher
+
+import (
+	"github.com/canaryio/canary"
+	"github.com/canaryio/canary/pkg/libratoaggregator"
+)
+
+type Publisher struct {
+	aggregator *libratoaggregator.Aggregator
+}
+
+func New(user, token string) *Publisher {
+	a := libratoaggregator.New(user, token)
+
+	return &Publisher{
+		aggregator: a,
+	}
+}
+
+func (p *Publisher) Start() {
+	go p.aggregator.Start()
+}
+
+func (p *Publisher) Publish(m canary.Measurement) (err error) {
+	// convert our measurement into a map of metrics
+	// send the map on to the librato aggregator
+	p.aggregator.C <- mapMeasurement(m)
+	return
+}
+
+// mapMeasurments takes a canary.Measurement and returns a map with all of the appriopriate metrics
+func mapMeasurement(m canary.Measurement) (metrics map[string]float64) {
+	// latency
+	// sampler  errors
+	// http errors
```

Comment:
def starting to feel that `canary.Measurement` should differentiate between HTTP and transport errors for us.  Might punt for this round though.